net-libs/nodejs/files/nodejs-16.5.0-add-riscv.patch

```diff
@@ -0,0 +1,967 @@
+From 1dd8202fb02cb3079d015c5d15c0433e455d63ce Mon Sep 17 00:00:00 2001
+From: Lu Yahan <yahan@iscas.ac.cn>
+Date: Mon, 17 Jan 2022 15:50:49 +0800
+Subject: [PATCH 1/5] deps: V8: cherry-pick d8dc66f92169
+
+Original commit message:
+
+    [riscv64][sparkplug] Fix sparkplug verify framesize failed
+
+    Change-Id: I7481749ba3d5c41d7405b0d88a51defbc8bec9d6
+    Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3093009
+    Auto-Submit: Yahan Lu <yahan@iscas.ac.cn>
+    Reviewed-by: Ji Qiu <qiuji@iscas.ac.cn>
+    Commit-Queue: Yahan Lu <yahan@iscas.ac.cn>
+    Cr-Commit-Position: refs/heads/master@{#76277}
+
+Refs: v8/v8@d8dc66f9
+---
+ common.gypi                                   |   2 +-
+ .../riscv64/baseline-assembler-riscv64-inl.h  | 260 +++++-------------
+ .../riscv64/baseline-compiler-riscv64-inl.h   |  64 +----
+ .../src/builtins/riscv64/builtins-riscv64.cc  |  68 +++--
+ .../codegen/riscv64/macro-assembler-riscv64.h |   2 +-
+ 5 files changed, 120 insertions(+), 276 deletions(-)
+
+diff --git a/common.gypi b/common.gypi
+index b8d61f0d3270..89fac788d3d2 100644
```

Comment:
Usually I like to drop these two lines as they don't really contain any useful data and make the patch larger overall (not needed to apply here if not convenient, just stating in general :) )